README.md

```diff
@@ -293,6 +298,8 @@ These installers will set automatically all the enviroment variables necessary t
 If you want to enable a [profile](#profile-cmake-options) or a [dependency](#dependencies-cmake-options) specific CMake option, you may need to install additional system dependencies following the dependency-specific documentation:
 * [`ROBOTOLOGY_USES_OCULUS_SDK`](#oculus)
 * [`ROBOTOLOGY_USES_CYBERITH_SDK`](#cyberith)
+* [`ROBOTOLOGY_USES_XSENS_SDK`](#xsens)
```

Comment:
In the documentation you always refer to the XSens SDK, but XSens is a company that provides several SDK, for example in https://github.com/robotology/xsensmt-yarp-driver we use the XSens MT SDK to read data from XSens MT* IMUs. To avoid confusion, could we refer to this as "XSens MVN SDK"? Or as anyway you find more suitable.

Reply:
done!